packages/block-library/src/paragraph/edit.js

```diff
@@ -52,6 +58,115 @@ function hasDropCapDisabled( align ) {
 	return align === ( isRTL() ? 'left' : 'right' ) || align === 'center';
 }
 
+function ParagrapgShortcuts( { attributes, clientId } ) {
+	const { align, content } = attributes;
+	const { replaceBlocks } = useDispatch( blockEditorStore );
+	const { registerShortcut } = useDispatch( keyboardShortcutsStore );
+	registerShortcut( {
+		name: 'core/blocks/paragraph/transform-to-heading-1',
+		category: 'block-library',
+		description: __( 'Transform to heading.' ),
```

Comment:
```suggestion
		description: __( 'Transform a paragraph to heading level 1.' ),
```
I think this should be a user facing description. Ideally it shows in the Keyboard shortcuts modal, but that's not happening for some reason. Possibly because the `category` isn't one that exists.